commands/build.go

```diff
@@ -496,6 +493,32 @@ func parseContextNames(values []string) (map[string]string, error) {
 	return result, nil
 }
 
+func writeMetadataFile(filename string, dt interface{}) error {
+	b, err := json.Marshal(dt)
```

Comment:
I guess removing `Marshalndent` was not intentional?

Reply:
Yes it was. I should have commented about that sorry. I think we should have the same behavior as `buildctl` but can revert that change if you want to.